semantic/src/site/elements/label.overrides

```diff
@@ -1,3 +1,8 @@
 /*******************************
          Site Overrides
 *******************************/
+
+.ui.label {
+  line-height: @lineHeightSmall;
```

Comment:
Just wondering -- I noticed you've used line-height in a few places. Do we really need it? In most cases the automatic line height is fine, so just wondering if you're going for something in particular?